### PR TITLE
Join table

### PR DESCRIPTION
**Pull request by AlaaShaheen2**
* In this project:

 *   I have done the following:
 *   Created a  table named vets.
 *   Created a join table named specializations.
 *  Created a join table named visits.
 *   Inserted data.
   -----------------------
* Write queries to answer the following

- [ ]     Who was the last animal seen by William Tatcher?
Screenshot 2022-11-05 at 0 06 19

- [ ]     How many different animals did Stephanie Mendez see?
Screenshot 2022-11-05 at 0 09 14

- [ ]     List all vets and their specialties, including vets with no specialties.
Screenshot 2022-11-05 at 0 11 29

- [ ]     List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
Screenshot 2022-11-05 at 0 12 38

- [ ]     What animal has the most visits to vets?
Screenshot 2022-11-05 at 0 14 11

- [ ]     Who was Maisy Smith's first visit?
Screenshot 2022-11-05 at 0 16 10

- [ ]     Details for most recent visit: animal information, vet information, and date of visit.
Screenshot 2022-11-05 at 0 18 47

- [ ]     How many visits were with a vet that did not specialize in that animal's species?
Screenshot 2022-11-05 at 0 20 22

- [ ]     What specialty should Maisy Smith consider getting? Look for the species she gets the most.
Screenshot 2022-11-05 at 0 21 26

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -61,4 +61,60 @@ SELECT name, full_name FROM animals JOIN owners ON animals.owner_id = owners.id
...
 
-SELECT full_name , COUNT(animals.name) AS all_animals FROM owners JOIN animals ON owner_id=owners.id GROUP BY owners.id ORDER BY all_animals DESC LIMIT 1;
\ No newline at end of file
+SELECT full_name , COUNT(animals.name) AS all_animals FROM owners JOIN animals ON owner_id=owners.id GROUP BY owners.id ORDER BY all_animals DESC LIMIT 1;
+
+--day4 project
+
+--Who was the last animal seen by William Tatcher?
+SELECT animal.id,  animal.name, vt.date_of_visit FROM animals 
+animal JOIN visits vt ON vt.animal_id =  animal.id JOIN vets v ON v.id = vt.vets_id WHERE v.name = 'William Tatcher' ORDER BY vt.date_of_visit DESC LIMIT 1;
```
Comment (jaflih) [inline queries.sql:70]: Please correct your request, you add animal twice : 

```
SELECT animal.id,  animal.name, vt.date_of_visit FROM **animals 
animal** JOIN visits vt ON vt.animal_id =  animal.id JOIN vets v ON v.id = vt.vets_id WHERE v.name = 'William Tatcher' ORDER BY vt.date_of_visit DESC LIMIT 1;
```
Correct request : 
```
SELECT animal.id,  animal.name, vt.date_of_visit FROM **animals ** JOIN visits vt ON vt.animal_id =  animal.id JOIN vets v ON v.id = vt.vets_id WHERE v.name = 'William Tatcher' ORDER BY vt.date_of_visit DESC LIMIT 1;
```

File: queries.sql [hunk 2]
```diff
@@ -61,4 +61,60 @@ SELECT name, full_name FROM animals JOIN owners ON animals.owner_id = owners.id
...
+
+--How many different animals did Stephanie Mendez see?
+SELECT COUNT(*) AS all_animals FROM animals 
+animal JOIN visits vt ON animal.id = vt.animal_id JOIN vets v on v.id = vt.vets_id WHERE v.id = 3;
```
Comment (jaflih) [inline queries.sql:74]: same here

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @AlaaShaheen2   

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
The Pull Request is following the GitHub Flow ✅
The Pull Request is well documented ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (AlaaShaheen2) [reply queries.sql:70]: @jaflih its not animal twice its animal and the  name of the table animals with s
its not the same

**Review by AshakaE — APPROVED** [review]
Hi @AlaaShaheen2 , 

### Highlights ✨ 

- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**